### PR TITLE
feat: Adjust Alert for CMS

### PR DESCRIPTION
**Pull request by israelswf**
## What's the purpose of this pull request?

This PR will replicate the changes made in this gatsby.store [PR](https://github.com/vtex-sites/gatsby.store/pull/29).

## How does it work?

The `Alert` (in the `src/common/Alert` folder) will get the properties `icon`, `text`, `linkText`, `actionLink` and `dismissible`, instead of just receiving everything as children (and hardcoded, like the `icon` and the `dismissible` properties)

## How to test it?

Nothing special to be tested, the `Alert` will be displayed exactly as before. It will just make the props available, so we can use them in a future PR in [storeframework.store](https://github.com/vtex-sites/storeframework.store). 

## References

Related [Jira task](https://vtex-dev.atlassian.net/browse/FSSS-304)

## Checklist

- [x] Added an entry in the `CHANGELOG.md` at the beginning of its due section. [The latest version should come first](https://keepachangelog.com/en/1.0.0/#:~:text=The%20latest%20version%20comes%20first.).
- [x] Added the PR number with the PR link at the entry in the `CHANGELOG.md`. E.g., *New items in the `pull_request_template.md` ([#4](https://github.com/vtex-sites/nextjs.store/pull/4))* 

- PR description
- [x] Added a label according to the PR goal - `Breaking change`, `Enhancement`, `Bug` or `Chore`.
- [x] Added the component, hook, or pathname in-between backticks (``) *- If applicable*. E.g., *`ComponentName` component*.

### CODE REVIEW

**Review by saranicoly — APPROVED** [review]

**Comment by filipewl** [comment]
Remember to add a new entry to the changelog.

File: src/components/common/Alert/Alert.tsx [hunk 1]
```diff
@@ -19,11 +28,12 @@ function Alert({ children }: PropsWithChildren<unknown>) {
...
     <UIAlert
-      icon={<Icon name="BellRinging" width={24} height={24} />}
-      dismissible
+      icon={<Icon name={icon} width={24} height={24} />}
+      dismissible={dismissible}
       onClose={onAlertClose}
+      link={link}
     >
-      {children}
+      {content}
```
Comment (ArthurTriis1) [inline src/components/common/Alert/Alert.tsx:36]: Why content and not children?

File: src/components/common/Alert/Alert.tsx [hunk 2]
```diff
@@ -1,20 +1,26 @@
...
+import type { ReactNode, PropsWithChildren } from 'react'
 
 import UIAlert from 'src/components/ui/Alert'
 import { mark } from 'src/sdk/tests/mark'
 import Icon from 'src/components/ui/Icon'
 
 interface Props {
   icon: string
-  content: ReactNode
+  content?: ReactNode
```
Comment (danzanzini) [inline src/components/common/Alert/Alert.tsx:10]: May we add a comment explaining what is the content for and how it behaves?

**Review by danzanzini — APPROVED** [review]
Nice. Just a minor comment

File: src/components/common/Alert/Alert.tsx (hunk 2 above)
Comment (lucasfp13) [reply src/components/common/Alert/Alert.tsx:14]: yesss, just did that [here](https://github.com/vtex-sites/nextjs.store/pull/31/commits/107f3b38cbcadb95574887afebb7019d34215706).

Comment (danzanzini) [reply src/components/common/Alert/Alert.tsx:14]: 🚀

**Pull request merged**